node/cli/src/command.rs

```diff
@@ -413,6 +413,36 @@ pub fn run() -> Result<()> {
 					.into())
 			}
 		}
+		#[cfg(feature = "try-runtime")]
+		/*
+		Some(Subcommand::CheckBlock(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			runner.async_run(|mut config| {
+				let (client, _, import_queue, task_manager) = service::new_chain_ops(&mut config)?;
+				Ok((cmd.run(client, import_queue), task_manager))
+			})
+		}
+		*/
```

Comment:
What's this commented code?

Reply:
It looks like my cherry-pick might have silently pulled that in, maybe? It looks like this block of code is duplicated below (without being commented).

Weird... I'll remove it, good catch!

Reply:
75e7cf6